netutils/ip.py

```diff
@@ -260,6 +385,32 @@ def is_netmask(netmask: str) -> bool:
         return False
 
 
+def is_network(ip_network: str) -> bool:
```

Comment:
I see a lot of overlap with `netaddr` , is this design decision not to use netaddr?

Reply:
correct
<img width="684" alt="image" src="https://github.com/networktocode/netutils/assets/9260483/10ebb068-6f17-422d-87d0-749debc2e359">